patches/000-initial-setup.patch

```diff
@@ -468,63 +592,65 @@ index 0000000000..4040c77bc1
 +
 +var ExtractHKDF = openssl.ExtractHKDF
 +var ExpandHKDF = openssl.ExpandHKDF
++var SupportsHKDF = openssl.SupportsHKDF
 diff --git a/src/crypto/rsa/pkcs1v15_test.go b/src/crypto/rsa/pkcs1v15_test.go
-index 69c509a771..05eac35d88 100644
+index 69c509a771..4383108679 100644
 --- a/src/crypto/rsa/pkcs1v15_test.go
 +++ b/src/crypto/rsa/pkcs1v15_test.go
-@@ -7,6 +7,7 @@ package rsa
+@@ -7,6 +7,8 @@ package rsa
  import (
  	"bytes"
  	"crypto"
 +	"crypto/internal/boring"
++	"crypto/internal/backend/boringtest"
  	"crypto/rand"
  	"crypto/sha1"
  	"crypto/sha256"
-@@ -52,6 +53,10 @@ var decryptPKCS1v15Tests = []DecryptPKCS1v15Test{
+@@ -52,6 +54,10 @@ var decryptPKCS1v15Tests = []DecryptPKCS1v15Test{
  }
  
  func TestDecryptPKCS1v15(t *testing.T) {
-+	if boring.Enabled {
++	if boring.Enabled && ! boringtest.Supports(t, "PKCSv1.5") {
 +		t.Skip("skipping PKCS#1 v1.5 encryption test with BoringCrypto")
 +	}
 +
  	decryptionFuncs := []func([]byte) ([]byte, error){
  		func(ciphertext []byte) (plaintext []byte, err error) {
  			return DecryptPKCS1v15(nil, rsaPrivateKey, ciphertext)
-@@ -76,6 +81,10 @@ func TestDecryptPKCS1v15(t *testing.T) {
+@@ -76,6 +82,10 @@ func TestDecryptPKCS1v15(t *testing.T) {
  }
  
  func TestEncryptPKCS1v15(t *testing.T) {
-+	if boring.Enabled {
++	if boring.Enabled && ! boringtest.Supports(t, "PKCSv1.5") {
```

Comment:
nit: Do you mind cleaning up the space in the negation operator? E.g. `!assertion()` as opposed to `! assertion()`.